src/shared/codelens/typescriptCodeLensProvider.ts

```diff
@@ -75,7 +75,7 @@ export function initialize({
             port: debugPort!,
             localRoot: samProjectCodeRoot,
             remoteRoot: '/var/task',
-            protocol: 'inspector',
+            protocol: params.runtime === 'nodejs6.10' ? 'legacy' : 'inspector',
```

Comment:
Also applies to versions older than 6.10

Reply:
I wan't thinking we were going to support end-of-life versions of node including node4 or earlier.

Reply:
If it shows up in our creation workflow, we implicitly support it. We've had some discussion around here about culling down the list to only "currently-supported" runtimes (https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html), but I'm not sure who the authority is on that--Lambda (where the code still runs), SAM (where templates can still be created), us, or someone else.